book/src/contributing/coding-standards.md

```diff
@@ -0,0 +1,3 @@
+## Coding Standards
+
+Coming Soon
```

Comment:
We should consider adopting this guide here: https://github.com/rust-lang/style-team/blob/master/guide/guide.md

Some have said it's the "official" style guide in the r/rust subreddit.

Reply:
Totally. I remember that link, I just didn't have the time to fully bake a coding standards section on the first pass.